pkg/syncer/syncers/downsyncer.go

```diff
@@ -0,0 +1,181 @@
+			ds.logger.Error(err, fmt.Sprintf("failed to get resource from upstream '%s'", resourceToString(resourceForUp)))
+			return err
+		}
+	}
+
+	resourceForDown := convertToDownstream(resource, conversions)
+	ds.logger.V(3).Info(fmt.Sprintf("  get '%s' from downstream", resourceToString(resourceForDown)))
+	downstreamResource, err := downstreamClient.Get(resourceForDown)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			// create
+			ds.logger.V(3).Info(fmt.Sprintf("  create '%s' in upstream since it's not found", resourceToString(resourceForDown)))
+			upstreamResource.SetResourceVersion("")
+			upstreamResource.SetUID("")
+			applyConversion(upstreamResource, resourceForDown)
+			if _, err := downstreamClient.Create(resourceForDown, upstreamResource); err != nil {
+				ds.logger.Error(err, fmt.Sprintf("failed to create resource to downstream '%s'", resourceToString(resourceForDown)))
+				return err
+			}
+		} else {
+			ds.logger.Error(err, fmt.Sprintf("failed to get resource from downstream '%s'", resourceToString(resourceForDown)))
+			return err
+		}
+	} else {
+		if downstreamResource != nil {
+			// update
+			ds.logger.V(3).Info(fmt.Sprintf("  update '%s' in upstream since it's found", resourceToString(resourceForDown)))
+			upstreamResource.SetResourceVersion("")
+			upstreamResource.SetUID("")
+			upstreamResource.SetManagedFields(nil)
+			applyConversion(upstreamResource, resourceForDown)
+			if _, err := downstreamClient.Update(resourceForDown, upstreamResource); err != nil {
+				ds.logger.Error(err, fmt.Sprintf("failed to update resource on downstream '%s'", resourceToString(resourceForDown)))
+				return err
+			}
+		} else {
+			msg := fmt.Sprintf("downstream resource is nil even if there is no error '%s'", resourceToString(resourceForDown))
+			return errors.New(msg)
+		}
+	}
+	return nil
+}
+
+func (ds *DownSyncer) BackStatusOne(resource edgev1alpha1.EdgeSyncConfigResource, conversions []edgev1alpha1.EdgeSynConversion) error {
+	upstreamClient, downstreamClient, err := ds.getClients(resource, conversions)
+	if err != nil {
+		ds.logger.Error(err, fmt.Sprintf("failed to get client '%s'", resourceToString(resource)))
+		return err
+	}
+	resourceForDown := convertToDownstream(resource, conversions)
+	downstreamResource, err := downstreamClient.Get(resourceForDown)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			ds.logger.V(3).Info(fmt.Sprintf("  not found '%s' in downstream", resourceToString(resourceForDown)))
+			ds.logger.V(3).Info(fmt.Sprintf("  skip status upsync '%s'", resourceToString(resourceForDown)))
+			return nil
+		} else {
+			ds.logger.Error(err, fmt.Sprintf("failed to get resource from upstream '%s'", resourceToString(resourceForDown)))
+			return err
+		}
+	}
+	status, found, err := unstructured.NestedMap(downstreamResource.Object, "status")
+	if err != nil {
+		ds.logger.Error(err, fmt.Sprintf("failed to extract status from downstream object '%s'", resourceToString(resourceForDown)))
+		return err
```

Comment:
Do you want a retry in this case?

Reply:
No... Maybe it would be better to have a flag in member variable of a resource client (/pkg/syncer/syncers/client.go) to indicate whether a resource has valid "status" field or not.